lib/client.js

```diff
@@ -36,9 +39,9 @@ function nop() {}
  * @param {Object} [options] - connection options
  * (see Connection#connect)
  */
-var MqttClient = module.exports =
-function MqttClient(streamBuilder, options) {
-  var that = this;
+MqttClient = module.exports = function MqttClient (streamBuilder, options) {
```

Comment:
This line is too long. Move `module.expors` at the bottom and remove the whole `var` thing.